static/src/stylesheets/amp/_d2-comments.scss

```diff
@@ -0,0 +1,50 @@
+/* Comments
+   ========================================================================== */
+
+.d2-comment-embedded {
+    background-color: #ffffff;
+    font-family: sans-serif;
+    overflow: hidden;
+    position: relative;
+    margin-bottom: 1.5em;
+    padding-top: 15px;
```

Comment:
Best to consult @zeftilldeath if it doesn't look right with grid values, it may be belong outside the grid. A note would be good if this is the case.

For the most part there has been a slightly simpler design on AMP to keep the css down. It is completely fine for the site and amp to have different styling as long as it still reflects the brand.